.gitignore

```diff
@@ -6,3 +6,6 @@ node_modules
 .env
 .env.*
 !.env.example
+/static/audio-worklet.bundle.js
```

Comment:
Pretty sure this was added by accident. The project no longer generates this bundle.

Reply:
fixd